pkg/webui/console/views/gateway/index.js

```diff
@@ -91,6 +91,7 @@ import PropTypes from '../../../lib/prop-types'
       'auto_update',
       'schedule_downlink_late',
       'update_channel',
+      'schedule_anytime_delay',
```

Comment:
Does it make sense to show this value also in the gateway overview? @htdvisser

Reply:
I don't have an opinion on this.

Reply:
Let's say no then.